api/enki.py

```diff
@@ -318,7 +323,11 @@ def _epoch_to_struct(cls, epoch_string):
     def checkout(self, patron, pin, licensepool, internal_format):
         identifier = licensepool.identifier
         enki_id = identifier.identifier
-        response = self.loan_request(patron.authorization_identifier, pin, enki_id)
+        enki_library_id = self.enki_library_id(patron.library)
+        response = self.loan_request(
+            patron.authorization_identifier, pin, enki_id,
+            enki_library_id
+        )
```

Comment:
Here the library ID does matter, quite a bit, so we grab it from the patron's library.